fvm/environment/minimum_required_version_test.go

```diff
@@ -0,0 +1,141 @@
+package environment
+
+import (
+	"testing"
+
+	"github.com/coreos/go-semver/semver"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_MapToCadenceVersion(t *testing.T) {
+	v0 := semver.Version{}
+	flowV1 := semver.Version{
+		Major:      1,
+		Minor:      2,
+		Patch:      3,
+		PreRelease: "rc.1",
+	}
+	flowV2 := semver.Version{
+		Major:      2,
+		Minor:      2,
+		Patch:      3,
+		PreRelease: "rc.1",
+	}
+
+	cadenceV1 := semver.Version{
+		Major:      2,
+		Minor:      1,
+		Patch:      3,
+		PreRelease: "rc.2",
+	}
+	cadenceV2 := semver.Version{
+		Major:      12,
+		Minor:      0,
+		Patch:      0,
+		PreRelease: "",
+	}
```

Comment:
```suggestion
	flowV1 := semver.Version{
		Major:      0,
		Minor:      37,
		Patch:      0,
	}
	flowV2 := semver.Version{
		Major:      0,
		Minor:      37,
		Patch:      1,
	}

	cadenceV1 := semver.Version{
		Major:      1,
		Minor:      0,
		Patch:      0,
	}
	cadenceV2 := semver.Version{
		Major:      1,
		Minor:      0,
		Patch:      1,
	}
```

Reply:
:+1: I changed the versions a little bit, because in the tests I bump the patch version and I dont want overlap